docs/KUBERNETES.md

```diff
@@ -136,27 +136,25 @@ Older versions of our configurations do not provide these labels. In that case,
 
 ### Upgrading the Image
 
-The image contains the actual agent code that is run on the Pods created by the DaemonSet. New versions of the agent always strive for backwards compatibility with old configuration versions. Any breaking changes will be outlined in the [release page](https://github.com/logdna/logdna-agent-v2/releases). We always recommend upgrading to the latest configuration to guarantee access to new features.
+The image contains the actual agent code that is run on the Pods created by the DaemonSet. New versions of the agent always strive for backwards compatibility with old configuration versions. Any breaking changes will be outlined in the [change log](https://docs.mezmo.com/changelog). We always recommend upgrading to the latest configuration to guarantee access to new features.
 
 The upgrade path for the image depends on which image tag you are using in your DaemonSet.
 
-If your DaemonSet is configured with `logdna/logdna-agent:stable`, the default configuration setting, then restarting your Pods will trigger them to pull down the latest stable version of the LogDNA agent image.
+If your DaemonSet is configured with `logdna/logdna-agent:3`, or some other major version number, then restarting your Pods will trigger them to pull down the latest minor version of the LogDNA agent image for this major version (in this example `3`).
```

Comment:
Nit: how about "...pull down the latest minor version for this major version (in this example `3`) of the LogDNA agent image."